Automattic/woocommerce-payments
# Merge Multi Currency Feature Branch into Development

#### Changes proposed in this Pull Request

This PR will merge the multi-currency feature branch into development. Currently, all functionality of this branch is behind a feature flag so this will initially have no customer-facing effect until we are ready to launch this feature.

#### Testing instructions

* Checkout this branch, ensure the `_wcpay_feature_customer_multi_currency` option is set to 0 in your local store.
* Do a quick smoke test and verify that all features of WC Pay work as expected and there is no reference to multi-currency in the `wp-admin` settings pages.
* Verify unit tests pass.

-------------------
